### PR TITLE
Kill Chrome quickly on Windows

### PR DESCRIPTION
**Pull request by JoelEinbinder**
This fixed the issues of profiles being left behind and errors deleting profiles still in use on Windows. Tested on Windows 10 in `cmd` and `cygwin`.

#298

### CODE REVIEW

File: lib/Launcher.js [hunk 1]
```diff
@@ -83,26 +83,32 @@ class Launcher {
 
     // Cleanup as processes exit.
     const listeners = [
-      helper.addEventListener(process, 'exit', killChromeAndCleanup),
-      helper.addEventListener(chromeProcess, 'exit', killChromeAndCleanup),
+      helper.addEventListener(process, 'exit', killChrome),
+      helper.addEventListener(chromeProcess, 'close', cleanupChrome),
```
Comment (JoelEinbinder) [inline lib/Launcher.js]: `close` is important to wait for all io to be done.

File: lib/Launcher.js [hunk 2]
```diff
@@ -83,26 +83,32 @@ class Launcher {
...
-    function killChromeAndCleanup() {
+    function cleanupChrome() {
       helper.removeEventListeners(listeners);
-      chromeProcess.kill('SIGKILL');
       removeSync(userDataDir);
     }
+
+    function killChrome() {
+      if (process.platform === 'win32')
+        childProcess.execSync(`taskkill /pid ${chromeProcess.pid} /T /F`);
```
Comment (JoelEinbinder) [inline lib/Launcher.js]: this is our heavy duty windows kill. Node reports `win32` for both 32 and 64 bit windows.

Comment (aslushnikov) [inline lib/Launcher.js]: Looks like chrome launcher, I like it :)

File: lib/Launcher.js [hunk 3]
```diff
@@ -83,26 +83,32 @@ class Launcher {
 
     // Cleanup as processes exit.
     const listeners = [
-      helper.addEventListener(process, 'exit', killChromeAndCleanup),
-      helper.addEventListener(chromeProcess, 'exit', killChromeAndCleanup),
+      helper.addEventListener(process, 'exit', killChrome),
```
Comment (aslushnikov) [inline lib/Launcher.js]: the `helper.removeEventListeners` is not needed any more. Could you please drop the `listeners` altogether and use the `once` methods?
```js
  chromeProcess.once('close', () => removeSync(userDataDir));
  process.once('exit', killChrome);
  if (options.handleSIGINT !== false)
    process.once('SIGINT', killChrome);
```

**Review by aslushnikov — APPROVED** [review]
Awesome!

File: lib/Launcher.js (hunk 3 above)
Comment (JoelEinbinder) [reply lib/Launcher.js]: once doesn't solve it completely because there are multiple paths into `killChrome`. I added a flag.

**Pull request merged**